### DISCUSSION TITLE
How to make any file open in the same tab when clicked from jupyter localhost window?

### DISCUSSION THREAD

**Discussion by duruburak** [jupyter/notebook#6740] [Q&A] [answered]
When I click any file (.ipynb, .txt, .jpg etc) from the jupyter localhost tree, it opens on another tab. I want it to be opened in the same tab, is it possible to do that?

**Comment by ccalvocm** [comment, marked as answer]
Greetings @duruburak 

Yes, it is possible to configure Jupyter Notebook to open files in the same tab instead of a new tab. Here's how you can do it:

Open your Jupyter Notebook configuration file. You can do this by running the following command in your terminal: jupyter notebook --generate-config.

Find the line that starts with c.NotebookApp.browser. If this line is commented out (i.e., it starts with a #), remove the # to uncomment it.

1. Add the following text to the end of the line: --new-window=False. This will tell Jupyter Notebook to open files in the same tab instead of a new tab.
2. Save and close the configuration file.
3. Restart your Jupyter Notebook server for the changes to take effect.

After completing these steps, when you click on a file in the Jupyter Notebook tree, it should open in the same tab instead of a new tab.

Or just use jupyter lab

**Comment by jtpio** [comment]
@duruburak you could also try using JupyterLab which allows for opening multiple notebooks in the same browser tab: https://jupyterlab.readthedocs.io/en/latest/
